### ISSUE TITLE
humidityStatus always undefined

### ISSUE THREAD

**Issue by ualex73** [maxwellhadley/node-red-contrib-rfxcom#87]
I installed version 2.9.0 and I noticed the humidity status is always "undefined". The humidity status works fine on my order home automation system and the packet is 100% identical, so it looks like a bug (I can't find the mapping of a value to a string like "comfort", "dry", "wet", etc in the code).

In the Node-RED debug I see the following:

```
23 Dec 09:50:08 - [info] [debug:4f67ea54.0c4ddc] 
{ topic: 'TH2/0xF605',
  status: { rssi: 6, battery: 9 },
  payload:
   { temperature: { value: 8, unit: 'degC' },
     humidity: { value: 81, unit: '%', status: undefined } },
  _msgid: 'd926095f.50fd78' }
```

**Comment by maxwellhadley** [comment]
You just found a very long-standing bug, well done! I have just published 2.9.1 which should fix this - please try it & let me know if it works now.

**Comment by ualex73** [comment]
Thanks! And I can confirm it fixed.
